### ISSUE TITLE
Process hangs indefinitely at "Creating a new placement group"

### ISSUE THREAD

**Issue by JoeyXuquant11** [huggingface/open-r1#617]
I don't know why it's been stuck on "No current placement group found. Creating a new placement group." for so long. I'm really frustrated.

Here is my script:

MODEL=deepseek-ai/DeepSeek-R1-Distill-Qwen-7B
NUM_GPUS=8
MODEL_ARGS="pretrained=$MODEL,dtype=bfloat16,data_parallel_size=$NUM_GPUS,max_model_length=32768,gpu_memory_utilization=0.8,generation_parameters={max_new_tokens:32768,temperature:0.6,top_p:0.95}" && \

if [ "aime24" = "lcb" ]; then \
        lighteval vllm $MODEL_ARGS "extended|lcb:codegeneration|0|0" \
                --use-chat-template \
                --output-dir data/evals/deepseek-ai/DeepSeek-R1-Distill-Qwen-32B; \
else \
        lighteval vllm $MODEL_ARGS "lighteval|aime24|0|0" \
                --use-chat-template \
                --output-dir data/evals/deepseek-ai/DeepSeek-R1-Distill-Qwen-32B; \
fi

\[2025-04-19 22:10:54,438] [    INFO]: PyTorch version 2.6.0 available. (config.py:54)
[2025-04-19 22:11:28,770] [INFO] [real_accelerator.py:222:get_accelerator] Setting ds_accelerator to cuda (auto detect)
INFO 04-19 22:11:35 [__init__.py:239] Automatically detected platform cuda.
[2025-04-19 22:11:42,278] [    INFO]: --- LOADING MODEL --- (pipeline.py:188)
[2025-04-19 22:11:43,787] [    INFO]: --- LOADING TASKS --- (pipeline.py:215)
[2025-04-19 22:11:43,788] [    INFO]: Found 1 custom tasks in /home/lyl/lighteval/src/lighteval/tasks/extended/ifeval/main.py (registry.py:141)
[2025-04-19 22:11:43,788] [    INFO]: Found 6 custom tasks in /home/lyl/lighteval/src/lighteval/tasks/extended/tiny_benchmarks/main.py (registry.py:141)
[2025-04-19 22:11:43,788] [    INFO]: Found 1 custom tasks in /home/lyl/lighteval/src/lighteval/tasks/extended/mt_bench/main.py (registry.py:141)
[2025-04-19 22:11:43,788] [    INFO]: Found 4 custom tasks in /home/lyl/lighteval/src/lighteval/tasks/extended/mix_eval/main.py (registry.py:141)
[2025-04-19 22:11:43,788] [    INFO]: Found 5 custom tasks in /home/lyl/lighteval/src/lighteval/tasks/extended/olympiade_bench/main.py (registry.py:141)
[2025-04-19 22:11:43,788] [    INFO]: Found 1 custom tasks in /home/lyl/lighteval/src/lighteval/tasks/extended/hle/main.py (registry.py:141)
[2025-04-19 22:11:43,788] [    INFO]: Found 21 custom tasks in /home/lyl/lighteval/src/lighteval/tasks/extended/lcb/main.py (registry.py:141)
[2025-04-19 22:11:43,790] [    INFO]: HuggingFaceH4/aime_2024 default (lighteval_task.py:187)
[2025-04-19 22:11:43,790] [ WARNING]: Careful, the task lighteval|aime24 is using evaluation data to build the few shot examples. (lighteval_task.py:260)
[2025-04-19 22:11:55,144] [    INFO]: --- INIT SEEDS --- (pipeline.py:261)
[2025-04-19 22:11:55,149] [    INFO]: --- RUNNING MODEL --- (pipeline.py:466)
[2025-04-19 22:11:55,149] [    INFO]: Running RequestType.GREEDY_UNTIL requests (pipeline.py:470)
[2025-04-19 22:11:55,161] [ WARNING]: You cannot select the number of dataset splits for a generative evaluation at the moment. Automatically inferring. (data.py:260)
Splits:   0%|                                                                                     | 0/1 [00:00<?, ?it/s][2025-04-19 22:11:55,174] [ WARNING]: context_size + max_new_tokens=33238 which is greater than self.max_length=32768. Truncating context to 0 tokens. (vllm_model.py:266)
[2025-04-19 22:12:01,343] [    INFO]: Started a local Ray instance. (worker.py:1841)
(pid=630802) INFO 04-19 22:12:26 [__init__.py:239] Automatically detected platform cuda.
(run_inference_one_model pid=630802) INFO 04-19 22:13:01 [config.py:600] This model supports multiple tasks: {'generate', 'score', 'reward', 'classify', 'embed'}. Defaulting to 'generate'.
(pid=630795) INFO 04-19 22:12:26 [__init__.py:239] Automatically detected platform cuda. [repeated 7x across cluster] (Ray deduplicates logs by default. Set RAY_DEDUP_LOGS=0 to disable log deduplication, or see https://docs.ray.io/en/master/ray-observability/user-guides/configure-logging.html#log-deduplication for more options.)
(run_inference_one_model pid=630795) INFO 04-19 22:13:01 [config.py:600] This model supports multiple tasks: {'generate', 'score', 'embed', 'classify', 'reward'}. Defaulting to 'generate'.
(run_inference_one_model pid=630835) INFO 04-19 22:13:01 [config.py:600] This model supports multiple tasks: {'embed', 'classify', 'generate', 'reward', 'score'}. Defaulting to 'generate'.
(run_inference_one_model pid=630835) INFO 04-19 22:13:01 [config.py:1780] Chunked prefill is enabled with max_num_batched_tokens=8192.
(run_inference_one_model pid=630828) INFO 04-19 22:13:01 [config.py:600] This model supports multiple tasks: {'reward', 'classify', 'generate', 'embed', 'score'}. Defaulting to 'generate'.
(run_inference_one_model pid=630818) INFO 04-19 22:13:01 [config.py:600] This model supports multiple tasks: {'generate', 'embed', 'reward', 'score', 'classify'}. Defaulting to 'generate'.
(run_inference_one_model pid=630819) INFO 04-19 22:13:01 [config.py:600] This model supports multiple tasks: {'reward', 'score', 'embed', 'generate', 'classify'}. Defaulting to 'generate'.
(run_inference_one_model pid=630805) INFO 04-19 22:13:01 [config.py:600] This model supports multiple tasks: {'classify', 'reward', 'score', 'embed', 'generate'}. Defaulting to 'generate'.
(run_inference_one_model pid=630808) INFO 04-19 22:13:01 [config.py:600] This model supports multiple tasks: {'reward', 'embed', 'generate', 'score', 'classify'}. Defaulting to 'generate'.
(run_inference_one_model pid=630795) INFO 04-19 22:13:05 [core.py:61] Initializing a V1 LLM engine (v0.8.3) with config: model='deepseek-ai/DeepSeek-R1-Distill-Qwen-7B', speculative_config=None, tokenizer='deepseek-ai/DeepSeek-R1-Distill-Qwen-7B', skip_tokenizer_init=False, tokenizer_mode=auto, revision=main, override_neuron_config=None, tokenizer_revision=main, trust_remote_code=False, dtype=torch.bfloat16, max_seq_len=32768, download_dir=None, load_format=LoadFormat.AUTO, tensor_parallel_size=1, pipeline_parallel_size=1, disable_custom_all_reduce=False, quantization=None, enforce_eager=False, kv_cache_dtype=auto,  device_config=cuda, decoding_config=DecodingConfig(guided_decoding_backend='xgrammar', reasoning_backend=None), observability_config=ObservabilityConfig(show_hidden_metrics=False, otlp_traces_endpoint=None, collect_model_forward_time=False, collect_model_execute_time=False), seed=1234, served_model_name=deepseek-ai/DeepSeek-R1-Distill-Qwen-7B, num_scheduler_steps=1, multi_step_stream_outputs=True, enable_prefix_caching=True, chunked_prefill_enabled=True, use_async_output_proc=True, disable_mm_preprocessor_cache=False, mm_processor_kwargs=None, pooler_config=None, compilation_config={"level":3,"custom_ops":["none"],"splitting_ops":["vllm.unified_attention","vllm.unified_attention_with_output"],"use_inductor":true,"compile_sizes":[],"use_cudagraph":true,"cudagraph_num_of_warmups":1,"cudagraph_capture_sizes":[512,504,496,488,480,472,464,456,448,440,432,424,416,408,400,392,384,376,368,360,352,344,336,328,320,312,304,296,288,280,272,264,256,248,240,232,224,216,208,200,192,184,176,168,160,152,144,136,128,120,112,104,96,88,80,72,64,56,48,40,32,24,16,8,4,2,1],"max_capture_size":512}
(run_inference_one_model pid=630835) INFO 04-19 22:13:05 [ray_utils.py:288] Ray is already initialized. Skipping Ray initialization.
(run_inference_one_model pid=630835) INFO 04-19 22:13:05 [ray_utils.py:335] No current placement group found. Creating a new placement group.

**Comment by kiseliu** [comment]
It seems a bug, see https://github.com/huggingface/lighteval/issues/670

**Comment by clefourrier** [comment]
Yep, it's an issue with ray - tldr is that there is a specific combo of ray + vllm versions which works and all the other combos hang or fail. Please follow the other issue for status on this :) (closing bc duplicate)
